IMC_Topic/IMC_name/Simple side-scrolling platformer game/proposal.md

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
This sounds really fun! Please make the following changes and create another PR - you should be good to go.
- Use the contentdevplan.md template in the devrel repo (linked in your issue)
- Remove all instance of curly braces